examples/truffle-migrate/index.js

```diff
@@ -8,6 +8,9 @@ async function main() {
   const value = await deployed.value();
   const version = await deployed.version();
   console.log(`Value of MyContract (${version}) deployed instance: ${value.toNumber()}`);
+  if(value != 52){
```

Comment:
```suggestion
  if (value !== 52) {
```

Reply:
Hi :) This PR adds a test to script in `index.js` that throws an error if the returned value is not the expected one.
fixes #767